elastic/kibana
# [ResponseOps][Alerting] Actions are not able to configure a max number of attempts

Resolves https://github.com/elastic/kibana/issues/79169

## Summary

The `ExecutorError` retry logic is working as expected. I updated the `getRetry()` function to retry based on attempts for unexpected errors that are thrown in `run()`


### Checklist



### To verify

- Create a rule with a server log action
- Add `maxAttempts: 3`, [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/actions/server/builtin_action_types/server_log.ts#L45)
- Add `throw new Error('fail');` in `x-pack/plugins/actions/server/lib/task_runner_factory.ts` somewhere in `run()`
- Verify that it will retry 3 times
